src/cmd/apply.ts

```diff
@@ -52,9 +52,13 @@ const commitOnFirstRun = async () => {
   }
   await waitTillAvailable(healthUrl)
 
-  if ((await nothrow($`git ls-remote`)).stdout.trim().length !== 0) return
+  if ((await nothrow($`git ls-remote`)).stdout.trim().length !== 0) {
```

Comment:
> If for some reason gitea is not respond (issue with ingress GW, DNS, service itself) then this script goes further.
> Can we have more reliable indicator of the fact that the initial deployment has succeeded?

We can't know from the perspective from this `apply` how charts/apps are doing after successful deployment. So what do you mean by `more reliable indicator of the fact that the initial deployment has succeeded`?

When we arrive at this point we _have_ to assume success up till now, so that is pretty binary, which leads me to the next comment:

> Instead of checking various resources, can we relay on config map which indicates status: (values-init, chart-init, helmfile-deploying, helmfile-deployed?

Let's avoid assumptions about different stages to tell different things (and those statements to be valuable), as there are not clear requirements, which goes against our principles (yagni). So either we come up with good use cases for that with clear requirements or we KISS for now. 

I suggest to keep it simple and just rely on one status that just says something like:
```yaml
status: deployed
deployedBy: cli | chart
```

Reply:
I like the fact of indicating tool that has deploying otomi.
Despite `deployed` status what other status values do you see?

Reply:
nothing else comes to mind actually so lets keep it at this